test/contracts/DynamicStringLength.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.4.2;
+import "./DynamicStringLengthCheck.sol";
+
+contract DynamicStringLength {
+  string public testString;
+  DynamicStringLengthCheck validate = new DynamicStringLengthCheck();
```

Comment:
I'm not sure I follow why this extra contract is necessary?

Reply:
The extra contract is there to validate the test string during contract-to-contract interaction.